### DISCUSSION TITLE
Will Controlnet-XS be available here?

### DISCUSSION THREAD

**Discussion by Njasa2k** [Mikubill/sd-webui-controlnet#2227] [Ideas]
ControlNet-XS: https://github.com/vislearn/ControlNet-XS

The models are 56.9MB for SD21 and 191MB for SDXL.

**Comment by ifsheldon** [comment]
Now diffusers 0.25 supports ControlNet-XS. Probably that implementation can be a reference.

**Comment by lavinal712** [comment]
Now ControlNet-XS is still not be supported by diffusers.

**Reply by huchenlei** [reply]
The published code was just so hard to reason about😅

**Reply by lavinal712** [reply]
I thought it was because ControlNet-XS does not meet “plug in and play now". Do you think ControlNet-XS is a effient way of controllable generation instead of ControlNet?
